import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/cstore/backup/BackupState.java

```diff
@@ -1,12 +1,12 @@
-package com.google.cloud.healthcare.imaging.dicomadapter.backupuploader;
+package com.google.cloud.healthcare.imaging.dicomadapter.cstore.backup;
 
 public class BackupState {
   private String uniqueFileName;
-  private int attemptsCountdown;
+  private volatile int attemptsCountdown = 1; //first send.
```

Comment:
I'm wondering why does this have to be volatile? Maybe we should copy BackupState instead of sharing one object to eliminate the need for volatile?

Reply:
it`s a part of fail logic that i have removed before. Fixed.